components/flowChart/FlowChart.jsx

```diff
@@ -52,6 +62,8 @@ const FlowChart = ({
 				properties: node.properties,
 				level: node.labels[0],
 				visibility: 'visible',
+				status: node.properties.final_controls_status, // Extract status
```

Comment:
Don't use application related terminologies in the root component. If you have used anywhere change.

Especially here, use `status` and `tag` instead of finalControlStatus and excluded resources